dotnet/Microsoft.MacCatalyst.Templates/maccatalyst/.template.config/template.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "MacCatalyst" ],
+  "identity": "Microsoft.MacCatalyst.MacCatalystApp",
+  "name": "MacCatalyst Application",
+  "description": "A project for creating an MacCatalyst application",
+  "shortName": "maccatalyst",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "sourceName": "MacCatalystApp1",
+  "preferNameDirectory": true,
+  "primaryOutputs": [
+    { "path": "MacCatalystApp1.csproj" }
+  ],
+  "symbols": {
+    "bundleId": {
+      "type": "parameter",
+      "description": "Overrides CFBundleIdentifier in the Info.plist",
```

Comment:
do we want to point out to the new .csproj/msbuild alternative ?
if later then we should file an issue so we don't forget about it

Reply:
We didn't do this for Android yet, because the IDE experience is weird when you use the new property.

I'll file an issue.